application.cpp

```diff
@@ -445,14 +462,18 @@ void application::exec() {
       (void)work;
       bool more = true;
       while( more || io_serv->run_one() ) {
+         if (my->_is_quiting)
+            break;
          while( io_serv->poll_one() ) {}
          // execute the highest priority item
          more = pri_queue.execute_highest();
       }
+      pri_queue.clear(); // make sure the queue is empty
 
       shutdown(); /// perform synchronous shutdown
    }
    io_serv.reset();
```

Comment:
Similar question as above; since we're about to destroy `application` we seemingly don't need to bother destroying `io_serv`.

Reply:
I agree, this `io_serv.reset();` os likely not needed.  I'll remove it.